gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystem.java

```diff
@@ -249,8 +250,14 @@ public GoogleHadoopFileSystem() {
       globalStorageStatistics = GhfsGlobalStorageStatistics.DUMMY_INSTANCE;
     }
 
-    GoogleCloudStorageEventBus.register(
-        new GoogleCloudStorageEventSubscriber(globalStorageStatistics));
+    globalStorageStatistics.incrementCounter(GoogleCloudStorageStatistics.GS_FILESYSTEM_CREATE, 1);
+  }
+
+  private static GhfsGlobalStorageStatistics createGlobalStatisticsSingleton() {
```

Comment:
1. this change will not allow `GhfsGlobalStorageStatistics.DUMMY_INSTANCE` to registered as eventbus subscriber..
2. why singleton in name? should `GhfsGlobalStorageStatistics` be changes to singleton class then?

Reply:
Please ignore this. This is just a temporary fix for the bug Gul is going to fix.